doc/source/development/contributing_environment.rst

```diff
@@ -36,29 +36,41 @@ These packages will automatically be installed by using the ``pandas``
 
 **Windows**
 
-You will need `Build Tools for Visual Studio 2019
+You will need `Build Tools for Visual Studio 2022
 <https://visualstudio.microsoft.com/downloads/>`_.
 
 .. warning::
-        You DO NOT need to install Visual Studio 2019.
-        You only need "Build Tools for Visual Studio 2019" found by
-        scrolling down to "All downloads" -> "Tools for Visual Studio 2019".
-        In the installer, select the "C++ build tools" workload.
+        You DO NOT need to install Visual Studio 2022.
+        You only need "Build Tools for Visual Studio 2022" found by
+        scrolling down to "All downloads" -> "Tools for Visual Studio".
+        In the installer, select the "Desktop development with C++" Workloads.
 
 You can install the necessary components on the commandline using
-`vs_buildtools.exe <https://download.visualstudio.microsoft.com/download/pr/9a26f37e-6001-429b-a5db-c5455b93953c/460d80ab276046de2455a4115cc4e2f1e6529c9e6cb99501844ecafd16c619c4/vs_BuildTools.exe>`_:
+`vs_buildtools.exe <https://download.visualstudio.microsoft.com/download/pr/f3f8db49-2cd0-43df-9ced-12dcb6b3954b/1a327074f0b07f951467739a34f07c5db6b491aa5df2ed3e4054759ceb1c994d/vs_BuildTools.exe>`_:
```

Comment:
Is this an alternative to the above? If so, how did you find the instructions, are they referenced somewhere? Just wondering if there's a link we could put instead, and this section is long (and will be hard to keep up-to-date)

Reply:
Yes, it is the new link to BuildTools for VisualStudio2022.

The instructions for installation are described at the [link](https://visualstudio.microsoft.com/downloads/) referenced above in the documentation. I received the detailed link by resolving the link behind Download button. The shorter form of the same link is `https://aka.ms/vs/17/release/vs_BuildTools.exe`. But I don't know which link will exist longer.

Agreed, I am not sure if we need to keep detailed instructions for the installation of the necessary components by using command-line.